src/openrct2-ui/input/KeyboardShortcuts.h

```diff
@@ -24,101 +24,106 @@
 
 struct ScreenCoordsXY;
 
-enum KeyboardShortcut
+enum class KeyboardShortcut
```

Comment:
There's already a class below with this exact name, but it's in the `OpenRCT2::Input` namespace, so it hints that this maybe should have a different name and also be moved there into the namespace.

Maybe this enum should become just shortcut, so that when it's moved there it becomes `OpenRCT2::Input::Shortcut`. What do you think @ZehMatt ?